### DISCUSSION TITLE
Not working on MacOS 15 Beta

### DISCUSSION THREAD

**Discussion by SPRTCS** [mrkai77/Loop#434] [Q&A]
Trigger Key and Shortcuts not Working on Beta 1 of MacOS 15

**Comment by mrkai77** [comment]
Can you explain further? I know others who have it running on macOS Sequoia without any problems, so it might be an issue on your side. Does Loop have accessibility access?

**Comment by SPRTCS** [comment]
Hello and thank you very much for your quick reply. I love you program and I’m using it on a daily basis. To be completely honest, I don’t know why it is not working. It seems to have a problem with the rights to see and modify applications on screen. I tried restarting the program and restarting my Mac, but the only thing that could fix the problem was removing it from the programs which can modify my screen and then re-adding it. that seems to be the workaround for now.macOS Sequoia is still asking me a few times a day if I still want to allow Loop to modify Windows on screen and I have to click on the allow button for it to continue function properly.Sorry for my bad English I am from Germany. Ich bedanke mich herzlich und verbleibe mit besten Grüßen!Arman SamaryLeiter Hochschulteam Berlin & KölnExecutive Financial ConsultantBachelor of Science Master Business AdministrationIHK Versicherungs- Darlehen & Finanzanlagenfachmann Mobil      +49 (0) 152 5891 ***@***.***@gmail.com  Persönliche AssistenzJana SchuhtMobil      +49 (0) 151 5190 ***@***.*** 20.06.2024 um 18:05 schrieb Kai ***@***.***>:﻿
Can you explain further? I know others who have it running on macOS Sequoia without any problems, so it might be an issue on your side. What are the indicators that it's not working?

**Reply by mrkai77** [reply]
Glad to hear that you found a fix! I will also be updating to macOS Sequoia shortly, so hopefully this bug will be gone soon :)
